src/components/transport_manager/src/bluetooth/bluetooth_transport_adapter.cc

```diff
@@ -57,13 +57,13 @@ CREATE_LOGGERPTR_GLOBAL(logger_, "TransportManager")
 BluetoothTransportAdapter::~BluetoothTransportAdapter() {}
 
 BluetoothTransportAdapter::BluetoothTransportAdapter(
-    resumption::LastState& last_state, const TransportManagerSettings& settings)
-    : TransportAdapterImpl(
-          new BluetoothDeviceScanner(this, true, 0, settings.bluetooth_uuid()),
```

Comment:
Why was this removed? `settings.bluetooth_uuid()`

Reply:
@jacobkeeler Thank you. It seems mistake of the last rebase. Reverted in 94f5c6bae6135e79bfb949aac37a7fbb7baf91d9